src/Model.js

```diff
@@ -184,6 +183,59 @@ const Model = class Model {
     }
 
     /**
+     * Update many-many relations for model.
+     * @param relations
+     */
+    _refreshMany2Many(relations) {
+        const ThisModel = this.getClass();
+        const This = this;
```

Comment:
I pushed the changes, the tests are green on the CI. Please have a look if its also working on your machine and feel free to rebase + fix them up if they do 🚀